netlify/functions/resources.ts

```diff
@@ -68,13 +68,45 @@ const resourcesHandler: Handler = async (event, context) => {
       body: tableToCsv(tableData),
     };
   } else if (resourceFilename === "findings.csv") {
-    const rawData = await readFile("./_data/findings/findings.csv", "utf8");
+    const findingsData = await getApiFindingsData();
+    let tableData: string[][] = [
+      [
+        "contest",
+        "handle",
+        "finding",
+        "risk",
+        "score",
+        "pie",
+        "split",
+        "slice",
+        "award",
+        "awardCoin",
+        "awardUSD",
+      ],
+      //contest,handle,finding,risk,score,pie,split,slice,award,awardCoin,awardUSD
+      ...findingsData.map((finding) => {
+        return [
+          "" + finding.contest,
+          finding.handle,
+          finding.finding,
+          finding.risk,
+          finding.score === null ? "" : "" + finding.score,
```

Comment:
ok, I can never remember the operator precedence of `?:` but looks like this works:

<img width="377" alt="testing the logic with findings score null vs. zero and getting intended results" src="https://user-images.githubusercontent.com/265902/233154721-26b67413-93f0-40eb-bb1b-a63e05f7612f.png">

still might feel "safer" as:

```suggestion
          finding.score === null ? "" : ("" + finding.score),
```

but up to you.

Reply:
[or the linter if it's `prettier` that wants it the current way!]